### PR TITLE
app-accessibility/epos: EAPI bump and patch updates

### PR DESCRIPTION
**Pull request by hanetzer**
Bump EAPI to 6, and cleanup the patches a bit

Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: app-accessibility/epos/epos-2.5.37-r2.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
 
-inherit eutils autotools
+inherit autotools
 
 DESCRIPTION="language independent text-to-speech system"
 HOMEPAGE="http://epos.ure.cas.cz/"
```
Comment (Moiman) [inline app-accessibility/epos/epos-2.5.37-r2.ebuild]: homepage doesn't work

File: app-accessibility/epos/epos-2.5.37-r2.ebuild [hunk 2]
```diff
@@ -18,14 +18,18 @@ DEPEND=">=app-text/sgmltools-lite-3.0.3-r9
...
-		"${FILESDIR}"/${P}-disable-tests.patch
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.5.37-gcc43.patch
+	"${FILESDIR}"/${PN}-2.5.37-gcc45.patch
+	"${FILESDIR}"/${PN}-2.5.37-gcc47.patch
+	"${FILESDIR}"/${PN}-2.5.37-disable-tests.patch
+)
 
+src_prepare() {
+	eapply "${PATCHES[@]}"
```
Comment (Moiman) [inline app-accessibility/epos/epos-2.5.37-r2.ebuild]: use default isntead

File: app-accessibility/epos/epos-2.5.37-r2.ebuild (hunk 1 above)
Comment (hanetzer) [reply app-accessibility/epos/epos-2.5.37-r2.ebuild]: Yeah, I think I may have noticed that in order to check what the software even is.
Not much I can do about that, however.

Comment (Moiman) [reply app-accessibility/epos/epos-2.5.37-r2.ebuild]: https://sourceforge.net/projects/epos/
http://epos.ufe.cz/

**Pull request closed without merging**